event/manager.go

```diff
@@ -161,20 +161,16 @@ func (m Manager) EventCategory(ctx context.Context, event types.BaseEvent) (stri
 }
 
 // Get the events from the specified object(s) and optionanlly tail the event stream
-func (m Manager) Events(ctx context.Context, objects []types.ManagedObjectReference, pageSize int32, tail bool, force bool, f func([]types.BaseEvent) error) error {
+func (m Manager) Events(ctx context.Context, objects []types.ManagedObjectReference, pageSize int32, tail bool, force bool, f func(types.ManagedObjectReference, []types.BaseEvent) error) error {
```

Comment:
I am going to leave this out for now. Got some other issues in VIC that I need to deal with.